hooks/pre/30-upgrade.rb

```diff
@@ -157,6 +187,7 @@ def fail_and_exit(message)
     upgrade_step :start_httpd, :run_always => true
     upgrade_step :start_qpidd, :run_always => true
     upgrade_step :start_pulp, :run_always => true
+    upgrade_step :upgrade_qpid_paths
```

Comment:
I dont know how the upgrade steps are ordered, but the `upgrade_qpid_paths` must be run when `qpidd` is stopped (so when it is recommended to have stopped also `pulp` or `candlepin` and so).